src/components/blog/post-card.tsx

```diff
@@ -12,11 +12,10 @@ import {
   Flex,
   Image,
   Badge,
-  Box
+  Box,
 } from "@chakra-ui/react";
 import { Link as NavLink } from "react-router-dom";
 import { FaEye } from "react-icons/fa";
-import { MotionBox } from "../motion";
 import { getTagColor } from "../../style/theme";
 import dev from "../../assets/images/logos/dev.png";
 import dev2 from "../../assets/images/logos/dev_white.png";
```

Comment:
@MA-Ahmad Although it's beyond the scope of these changes; I think it is a good idea to set a path alias so you can simple refer from the root directory rather than traversing to others above the current location.

e.g.

```diff
-import dev from "../../assets/images/logos/dev.png";
-import dev2 from "../../assets/images/logos/dev_white.png";
+import dev from "@/assets/images/logos/dev.png";
+import dev2 from "@/assets/images/logos/dev_white.png";
```

where `@/` is from `src/`

We can do this by extending `tsconfig.json`

Rough reference: https://stackoverflow.com/questions/57070052/create-react-app-typescript-3-5-path-alias